clmm/theory/parent_class.py

```diff
@@ -441,6 +448,22 @@ def get_einasto_alpha(self, z_cl=None):
             raise ValueError(f"Wrong profile model. Current profile = {self.halo_profile_model}")
         return self._get_einasto_alpha(z_cl)
 
+    def set_projected_quad(self, projected_quad):
```

Comment:
can you change `projected_quad` to something like `use_projected_quad`?
Also do this for `_set_projected_quad` functions